packages/library/src/airdrop.ts

```diff
@@ -18,10 +22,22 @@ type AirdropFactoryConfig = Readonly<{
 }>;
 
 export function airdropFactory({ rpc, rpcSubscriptions }: AirdropFactoryConfig): AirdropFunction {
-    const confirmSignatureOnlyTransaction = createDefaultSignatureOnlyRecentTransactionConfirmer({
+    const getRecentSignatureConfirmationPromise = createRecentSignatureConfirmationPromiseFactory(
         rpc,
         rpcSubscriptions,
-    });
+    );
+    async function confirmSignatureOnlyTransaction(
+        config: Omit<
+            Parameters<typeof waitForRecentTransactionConfirmationUntilTimeout>[0],
+            'getRecentSignatureConfirmationPromise' | 'getTimeoutPromise'
+        >,
+    ) {
+        await waitForRecentTransactionConfirmationUntilTimeout({
+            ...config,
+            getRecentSignatureConfirmationPromise,
+            getTimeoutPromise,
+        });
+    }
```

Comment:
Isn't this marked as deprecated?

Reply:
I think Steven removed its deprecated status because tests were added for the `waitForRecentTransactionConfirmationUntilTimeout` function. Any reason it was deprecated before @steveluscher?

Reply:
Oh actually I can still see it as `@deprecated` on the new package. I'm confused.

Reply:
It's definitely deprecated, but there's no other way to confirm airdrops, since the `requestAirdrop` RPC call doesn't return the `lastValidBlockHeight`.